go.mod

```diff
@@ -17,7 +17,7 @@ require (
 	github.com/nspcc-dev/locode-db v0.6.0
 	github.com/nspcc-dev/neo-go v0.105.1
 	github.com/nspcc-dev/neofs-api-go/v2 v2.14.1-0.20240305074711-35bc78d84dc4
-	github.com/nspcc-dev/neofs-contract v0.19.2-0.20240427112531-286b14199fd0
+	github.com/nspcc-dev/neofs-contract v0.19.2-0.20240506202632-e78d64ecdfc2
```

Comment:
can you specify this update somehow in the commit's message? usually, i prefer to mention updating a dep as a commit's reason to exist and some usage bonuses rather than fixing something and absolutely no info about updating a dep

Reply:
It's _so_ internal and we'll be updating it again.

Reply:
Fixed though.